se2_planning/src/OmplPlanner.cpp

```diff
@@ -35,6 +35,10 @@ void OmplPlanner::getPath(Path* path) const {
 bool OmplPlanner::plan() {
   simpleSetup_->clear();
   simpleSetup_->setStartAndGoalStates(*startState_, *goalState_);
+  // TODO see https://ompl.kavrakilab.org/genericPlanning.html for continue planning
+  //   The ompl::base::Planner::solve() method can be called repeatedly with different
```

Comment:
I would leave this a bonus for now. If I understand correctly it is not necessary for correct functioning of the algorithm.

Reply:
Yes, this might be an option to speed things up. Not required.